### PR TITLE
Replace SkosFeedable concern with SchemeDefinitionFetchable concern

### PR DESCRIPTION
**Pull request by richard0000**
# What was done?
- This new concern fetches the definition using the provided URI, then
- With the content creates a temp file with the corresponding extension
- If the extension can't be inferred it will default to JSON
- If it can't be parsed, it will throw a proper error.

# Why?
The format converter was only used before when a user meant to upload a file. Now with configuration profiles, when it's activated we should use the declared URI's for schemes, abstract classes, predicates, and concept schemes to transform each of those into JSON-LD which is the format that the tool handles.

### CODE REVIEW

File: app/models/concerns/scheme_definition_fetchable.rb [hunk 1]
```diff
@@ -35,6 +31,20 @@ def resolve_context
...
   private
 
+  def temp_file uri
+    file = Tempfile.new(["tmpschemfile", infer_extension(uri)])
+    file.write(fetch_content(uri).force_encoding("UTF-8"))
+    file.rewind
+    file
+  end
+
+  def fetch_content uri
```
Comment (excelsior) [inline app/models/concerns/scheme_definition_fetchable.rb:41]: Hmm, strangely enough, this addition made the tests ~60% slower on my machine (8m vs 5m). I guess we can investigate this regression later.

**Review by excelsior — APPROVED** [review]

File: app/models/concerns/scheme_definition_fetchable.rb (hunk 1 above)
Comment (richard0000) [reply app/models/concerns/scheme_definition_fetchable.rb:41]: Agree. I'll create an issue for this then.

Comment (richard0000) [reply app/models/concerns/scheme_definition_fetchable.rb:41]: Issue created [#154](https://github.com/t3-innovation-network/desm/issues/154).

**Comment by stuartasutton** [comment]
@richard0000, we spoke briefly in our last meeting about having a dropdown with expected file extensions: cvs, json, json-ld , xml, rdf (i.e. rdf/xml), ttl thus being explicit and not needing to infer.

**Pull request merged**